### PR TITLE
feat - tbfz game <3

### PR DESCRIPTION
**Pull request by franRappazzini**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @franRappazzini,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/340) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

**Comment by jessiemongeon1** [comment]
Hi @franRappazzini , please resolve the conflicts within the `showcase.json` file. Thanks!

**Comment by franRappazzini** [comment]
Sorry, I had not seen. That's it! Thank you!

File: showcase.json [hunk 1]
```diff
@@ -2402,6 +2402,23 @@
     "videoContentType": "video/mp4",
     "submittableId": "33308492"
   },
+  {
+    "id": "tbfz",
+    "name": "The Battle for Zhion",
+    "website": "https://tbfz.io/",
+    "oneLiner": "Survive, conquer, and master the power of Zhion's battleground.",
+    "tags": ["Games", "Metaverse", "NFT", "Bitcoin", "Wallet"],
```
Comment (letmejustputthishere) [inline showcase.json:2410]: can you please remove the Bitcoin and Wallet tags or explain why they are relevant?

**Review by letmejustputthishere — CHANGES_REQUESTED** [review]

**Comment by franRappazzini** [comment]
@letmejustputthishere ready! thank you!

**Review by jennifertrin — APPROVED** [review]

**Review by jennifertrin — CHANGES_REQUESTED** [review]
Hello @franRappazzini, apologies, can you please follow the CLA steps above?

**Review by jennifertrin — APPROVED** [review]

**Comment by jennifertrin** [comment]
@franRappazzini Stoll advised that you will open a PR. Please open a new PR at your earliest convenience.

**Pull request closed without merging**